app/controllers/ops_controller/ops_rbac.rb

```diff
@@ -902,13 +902,45 @@ def rbac_group_get_details(id)
   def rbac_group_right_tree(selected_nodes)
     case @sb[:active_rbac_group_tab]
     when 'rbac_customer_tags'
-      @tags_tree = TreeBuilderTags.new(:tags_tree,
-                                       :tags,
-                                       @sb,
-                                       true,
-                                       :edit    => @edit,
-                                       :filters => @filters,
-                                       :group   => @group)
+      cats = Classification.categories.find_all do |c|
+        c if c.show || !%w[folder_path_blue folder_path_yellow].include?(c.name)
```

Comment:
The `Enumerable#find_all` expects a boolean from the block, therefore, I don't understand why you want to use an `if` inside and return with the data itself. Also a `select` or a `reject` might be more understandable.